line-sdk/build.gradle

```diff
@@ -122,6 +135,39 @@ ext {
     ]
 }
 
+task javadoc_public(type: Javadoc) {
+    group javadocParams.taskGroup
+    destinationDir = javadocParams.destDir.public
+
+    // doclava doesn't support `title`
+    title = null
+
+    source = javadocParams.source
+    classpath += javadocParams.classpath
+    include javadocParams.includes.public
+
+    failOnError false
+
+    /*
+    options {
+        doclet "com.google.doclava.Doclava"
+        docletpath = configurations.doclava.files.asType(List)
+
+        locale javadocParams.locale
+        encoding javadocParams.encoding
+
+        // doclava supports `-title ${title}`
+        addStringOption("title", "${javadocParams.title}")
+        addStringOption("templatedir", "${rootDir}/templates")
+        addMultilineMultiValueOption("federate").with {
+            javadocParams.externalLinks.each {
+                value.add(["name", it])
+            }
+        }
+    }
+     */
```

Comment:
Why do we comment out this? If the code is not necessary, we can remove it. If we need to do something in the future, it's better to add a TODO comment to explain it.

Reply:
comments added.